helm/node-normalization-loader/values.yaml

```diff
@@ -60,61 +60,61 @@ data:
         redis_db: "gene_protein_db"
 codeDir: /code
 
-redis_backend_config:
-  "eq_id_to_id_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      # list of cluster member ips and ports
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
-
-  "id_to_eqids_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
-
-  "id_to_type_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
-
-  "curie_to_bl_type_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
-
-  "gene_protein_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
-
-  "info_content_db":
-    "ssl_enabled": false
-    "is_cluster": false
-    "db": 0
-    "hosts":
-      - "host_name": "host"
-        "port": "6379"
-    "password": "password"
+redis_backend_config: {}
```

Comment:
did you actually want to save this commented out bit here?

Reply:
I left it for reference but we sure can remove it since we have it in the other values files